extensions/BMO/template/en/default/pages/etiquette.html.tmpl

```diff
@@ -18,133 +18,147 @@
+  </li>
+  
+  <li>
+    <span class="heading">No spam</span>.
+    Posting comment spam will lead to the suspension of your account.
   </li>
 
   <li>
-  <span class="heading">No abusing people</span>.
-  Constant and intense critique is one of the reasons we build great products.
-  It's harder to fall into group-think if there is always a healthy amount of
-  dissent. We want to encourage vibrant debate inside of the Mozilla
-  community, we want you to disagree with us, and we want you to effectively
-  argue your case. However, we require that in the process, you attack
-  <i>things</i>, not <i>people</i>. Examples of things include: interfaces,
-  algorithms, and schedules. Examples of people include: developers,
-  designers and users. <b>Attacking a person may result in you being banned
-  from [% terms.Bugzilla %].</b>
+    <span class="heading">No pointless comments</span>.
+    Limit comments on a [% terms.bug %] to information which will help with 
+    resolving it. Unless requested, additional "I see this too" or "It works for me" 
+    comments are unnecessary. Constructive conversations unrelated to the topic of 
+    the [% terms.bug %] should go in the appropriate
+     <a href="https://www.mozilla.org/about/forums/">discussion forum</a>.
   </li>
 
   <li>
-  <span class="heading">No private email</span>.
-  Unless the [% terms.bug %] owner or another respected project contributor has asked you
-  to email them with specific information, please place all information
-  relating to [% terms.bugs %]
-  in the [% terms.bug %] itself. Do not send them by private email; no-one else can read
-  them if you do that, and they'll probably just get ignored. If a file
-  is too big for [% terms.Bugzilla %], add a comment giving the file size and contents
-  and ask what to do.
+    <span class="heading">No private email</span>.
+    Do not send comments on [% term.bugs %] by private email to users; 
+    no one else can read them if you do that, and they'll be missed and/or ignored.
+    If an attachment is too big for [% terms.Bugzilla %], add a comment 
+    giving the file size and contents and ask what to do.
   </li>
 </ol>
 
-<h3>2. Changing Fields</h3>
+<h3>Changing Fields</h3>
 
 <ol>
   <li>
-  <span class="heading">No messing with other people's [% terms.bugs %]</span>.
-  Unless you are the [% terms.bug %] assignee, or have some say over the use of their
-  time, never change the Priority or Target Milestone fields. If in doubt,
-  do not change the fields of [% terms.bugs %] you do not own - add a comment
-  instead, suggesting the change.
+    <span class="heading">No messing with other people's [% terms.bugs %]</span>.
+    Unless you are the [% terms.bug %] assignee, or have some say over the use of their
+    time, never change the Priority or Target Milestone fields. If in doubt,
+    do not change the fields of [% terms.bugs %] you do not own—add a comment
```

Comment:
@markrcote may be right in standard typography, but omitting spaces around an em-dash just leads people to confuse it with a hyphenated word. In books where this is done, an em-dash is very long. In most ordinary fonts, it's not long enough. I certainly find my reading hiccuping whenever I see this. The ultimate aim of punctuation is an aid to readability, and this construction fails that test.